### PR TITLE
Standardise outputs for video pipelines

### PR DESCRIPTION
**Pull request by DN6**
# What does this PR do?
This PR 
1. Standardises the output for the following pipelines by making them all use the same `tensor2vid` function to postprocess the 3D tensor output with shape (batch size, channels, num frames, height, width). 

TextToVideoSDPipeline
AnimateDiff
StableVideoDiffusionPipeline

2. Updates TextToVideoSDPipeline tests so that shapes of expected numpy array match the output from new postprocessing. 

TextToVideoZeroPipeline and TextToVideoZeroSDXLPipeline are excluded because they make use of 2D UNets, and produce an output with shape (batch size, channels, height, width) where the batch size == number of frames generated.  

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/diffusers/pipelines/animatediff/pipeline_animatediff.py [hunk 1]
```diff
@@ -68,9 +68,6 @@
 
 
 def tensor2vid(video: torch.Tensor, processor, output_type="np"):
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/animatediff/pipeline_animatediff.py:70]:
```suggestion
def tensor2vid(video: torch.Tensor, processor: VaeImageProcessor, output_type: str = "np"):
```

File: src/diffusers/pipelines/animatediff/pipeline_animatediff.py [hunk 2]
```diff
@@ -79,6 +76,12 @@ def tensor2vid(video: torch.Tensor, processor, output_type="np"):
 
         outputs.append(batch_output)
 
+    if output_type == "np":
+        outputs = np.stack(outputs)
+
+    if output_type == "pt":
+        outputs = torch.stack(outputs)
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/animatediff/pipeline_animatediff.py:83]:
```suggestion
    if output_type == "np":
        outputs = np.stack(outputs)
    elif output_type == "pt":
        outputs = torch.stack(outputs)
    else:
        raise ValueError(f"{output_type} does not exist. Please choose one of ['np', 'pt']")
```

Or should we also allow returning a list of PIL images?

Comment (patrickvonplaten) [reply src/diffusers/pipelines/animatediff/pipeline_animatediff.py:83]: If yes we could maybe change:
```diff
-     else:
+     elif not output_type == "pil":
```

File: src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py [hunk 3]
```diff
@@ -40,10 +40,8 @@ def _append_dims(x, target_dims):
     return x[(...,) + (None,) * dims_to_append]
 
 
+# Copied from diffusers.pipelines.animatediff.pipeline_animatediff.tensor2vid
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py:43]: Nice!

**Review by patrickvonplaten — APPROVED** [review]
Nice clean-up!

File: src/diffusers/pipelines/animatediff/pipeline_animatediff.py (hunk 2 above)
Comment (DN6) [reply src/diffusers/pipelines/animatediff/pipeline_animatediff.py]: We do support returning a PIL list as well. Will update.

**Pull request merged**